spec/integration/manifest.spec.yml

```
@@ -410,6 +410,33 @@ spec:
                 - access
           deployment_modes:
             $ref: "#/definitions/deployment_modes"
+          links:
+            description: List of links related to policy template.
+            type: array
+            minItems: 1
+            items:
+              type: object
+              additionalProperties: false
+              properties:
+                title:
+                  description: Link title
+                  type: string
+                url:
+                  description: Link url
+                  type: string
+                type:
+                  description: Type of link. `next_steps` for a regular link, `action` for a link button.
+                  type: string
+                  enum:
+                    - action
+                    - next_steps
```

Nit. Maybe we should be consistent with the use of plural here.
```suggestion
                    - action
                    - next_step
```